### ISSUE TITLE
Logs Seem To Build Up On Disk

### ISSUE THREAD

**Issue by ChristianSilvermoon** [Wohlstand/TheXTech#13]
Using TheXTech for about 6 days has built up a total of 43 different log files on the drive totally 6 MB in size.

6 MB isn't too terribly much in the short term, but if I continued playing the game frequently (as I probably will) the logging will add up.

using `gzip -9` on the logs brought the size down to 280 KB, which could help.
If only the latest log were uncompressed, this could significantly cut down on space consumption.
Additionally it would make it extremely easy to spot the latest log.

To prevent even too much space usage by even compressed excess logs, perhaps an option in `TheXTech.ini` along the lines of `max-log-count` for how many logs should be kept, defaulting to a number like 10?

tl;dr Consider limiting the maximum number of logs TheXTech stores and possibly compressing the previous log on startup.

**Comment by Wohlstand** [comment]
Good idea! Since PGE Engine which was WIP, I had no care about logs count, and I though once upon I will need to optimize that... Thanks for a note!

**Comment by ChristianSilvermoon** [comment]
Of course ^^

**Comment by Wohlstand** [comment]
Done: https://github.com/Wohlstand/TheXTech/commit/184f40fbc6fbf59e024a64355eb5199c845dc971

**Comment by ChristianSilvermoon** [comment]
184f40f seems to work perfectly, but creates up 11 logs instead of 10 though x3

**Comment by Wohlstand** [comment]
Whoops, looks like a small computing mistake, easy to fix

**Comment by Wohlstand** [comment]
Yeah! Why 11? Because I cleaning up logs before to open new. These 10 are kept correctly, but no count of that new 11th that will be created later 🤔

**Comment by ChristianSilvermoon** [comment]
Ah that makes sense, yeah.

**Comment by Wohlstand** [comment]
Just now I had to fix the count of logs, so, should be fine.
